docs/content/basic/authenticating-oauth.md

```diff
@@ -4,256 +4,703 @@ lang: en
 slug: /concepts/authenticating-oauth
 ---
 
-To prepare your Slack app for distribution, you will need to enable Bolt OAuth and store installation information securely. Bolt supports OAuth and will handle the rest of the work; this includes setting up OAuth routes, state verification, and passing your app an installation object which you must store. 
+OAuth allows installation of your app to any workspace and is an important step
+in distributing your app. For each completed authentication process, you receive
+unique [access tokens and related information](#the-installation-object) that
+can be retrieved for incoming events and used to make scoped API requests.
 
-To enable OAuth, you must provide:
-* `clientId`, `clientSecret`, `stateSecret` and `scopes` _(required)_
-* An `installationStore` option with handlers that store and fetch installations to your database *(optional, strongly recommended in production)*
+All of the additional underlying details around authentications can be found at
+[api.slack.com/authentication/oauth-v2][oauth-v2]!
 
----
+## Configuring the application
+
+To set your Slack app up for distribution, you will need to enable Bolt OAuth
+and store installation information securely. Bolt supports OAuth by using
+[`@slack/oauth`][oauth-node] to handle most of the work; this includes setting
```

Comment:
```suggestion
the [`@slack/oauth`][oauth-node] package to handle most of the work; this includes setting
```

or... "library"? not sure what is proper. But we like to say what the thing is instead of just saying the thing. Example: instead of saying "use `conversations.info` to do it" it should be "use the `conversations.info` method to do it"

Reply:
🧠 Excellent practice here- I'm in quick agreement! Thanks for finding these fixes 🙏

I'll keep `eye` on myself for this one 😅